skcosmo/decomposition/pcovr.py

```diff
@@ -432,11 +432,11 @@ def _fit_sample_space(self, X, Yhat, W):
             self.explained_variance_ / self.explained_variance_.sum()
         )
 
-        P = (self.mixing * X.T) + (1.0 - self.mixing) * np.dot(W, Yhat.T)
+        P = (self.mixing * X.T) + (1.0 - self.mixing) * W @ Yhat.T
```

Comment:
Should this not be `Y`?

Reply:
I don't think so. The way `P` is derived is to pull an `X` out of the `X @ X.T` and `Yhat @ Yhat.T` terms, which leaves `X.T` and `W @ Yhat.T`, respectively

Reply:
Ah yes, forgetting the context.